presto-orc/src/main/java/com/facebook/presto/orc/metadata/DwrfMetadataReader.java

```diff
@@ -91,7 +91,8 @@ public PostScript readPostScript(byte[] data, int offset, int length)
         return new PostScript(
                 ImmutableList.of(),
                 postScript.getFooterLength(),
-                0,
+                postScript.getCacheSize(),
```

Comment:
Is `metadataLength` same as `cacheSize`? Worth a comment to clarify?

Reply:
It's a bit different. ORC has metadata, but doesn't have the stripe metadata cache; DWRF doesn't have metadata, but has the stripe metadata cache. It was convenient to re-use the metadata fields because these formats are mutually exclusive in this area, but it's better to separate them. 

I'll post an update to this PR with clear separation of metadata and stripe cache.